store/iavl/store_test.go

```diff
@@ -667,6 +667,7 @@ func TestCacheWraps(t *testing.T) {
 }
 
 func TestChangeSets(t *testing.T) {
+	t.Skip("This test causes memdb to deadlock in CI, in a running network a non in memory db will be used")
```

Comment:
imho, best to temporarily have a super high flush threshold with something like this: `tree := iavl.NewMutableTree(db, cacheSize, false, log.NewNopLogger(), iavl.FlushThresholdOption(math.MaxInt))` than the test altogether.

Reply:
good call, added